docs/src/managing-packages.md

```diff
@@ -81,7 +81,10 @@ we can explicitly track that branch (or commit) by appending `#branchname` (or `
 ```
 
 The status output now shows that we are tracking the `master` branch of `Example`.
-When updating packages, we will pull updates from that branch.
+When updating packages, we will pull updates from that branch. Notice that if instead
+of a branch name we specified an exact commit id, e.g.
+`add Example#277827f2b18e14a611d98144422eb29f61498104`, then we would be effectively
```

Comment:
This commit SHA is excessively long. I can't just edit it myself because the repo-fork this PR is made from has been deleted.

Reply:
Ops. I'll try to fix it. Why don't you type the text you want here in the meantime, and I'll get it going.